docker-compose.yaml

```
@@ -0,0 +1,11 @@
+version: '3.8'
+
+services:
+  switchmap-ng-web: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)